src/AssemblySharedInfoGenerator/AssemblySharedInfo.tt

```diff
@@ -9,7 +9,7 @@ using System.Runtime.InteropServices;
 // associated with an assembly.
 [assembly: AssemblyCompany("Autodesk, Inc")]
 [assembly: AssemblyProduct("Dynamo")]
-[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2014")]
+[assembly: AssemblyCopyright("Copyright © Autodesk, Inc 2015")]
 [assembly: AssemblyTrademark("")]
```

Comment:
Since this is a merge to the Revit2014 branch I think this should remain as 2014